coalib/settings/ConfigurationGathering.py

```diff
@@ -240,9 +240,15 @@ def load_configuration(arg_list,
             Setting('config', re.escape(find_user_config(os.getcwd()))))
 
     targets = []
+    EXCLUDE_PREFIX = '!'
```

Comment:
just thinking: is there maybe a place where we can store this constant except here? It's also okay here, but maybe in coala there's a more appropriate place^^ (like `Constants.py`)
Up to you to decide that ;)